extra-network/v2ray-desktop/autobuild/beyond

```diff
@@ -0,0 +1,5 @@
+install -Dm644 "$PKGDIR"/opt/V2Ray-Desktop/bin/V2Ray-Desktop "$PKGDIR"/usr/bin/v2ray-desktop
```

Comment:
Please use verbosity throughout, e.g. `install -Dvm644` and `rm -rv` instead of `install -Dm644` and `rm -r`. Also please add `abinfo` at each step to indicate script progression.

I'd also suggest "chopping" commands with a `\` at the end of the first operand to shorten the lines.

Please review our [AOSC OS Package Styling Manual](https://wiki-aosc.netlify.app/developer/packaging/package-styling-manual/) before your next revision.